### PR TITLE
Support man page formatting

### PR DESCRIPTION
**Pull request by aweiteka**
Provide basic formatting markup so md2man renders properly.

@mtrmac please review

### CODE REVIEW

File: signature/policy_types.go [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Note: Consider the API unstable until the code supports at least three different image formats or transports.
+//       Keep this in sync with documentation
```
Comment (mtrmac) [inline signature/policy_types.go]: `policy_types.go` has nothing to do with `registries.d`. The relevant comment already is in https://github.com/containers/image/blob/master/docker/lookaside.go#L28 .

File: docs/registries.d.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
-<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
-
+% REGISTRIES.D(5) Registries.d Man Page
+% Miloslav Trmac
```
Comment (mtrmac) [inline docs/registries.d.md]: `Trmač` , please (in both places)

**Comment by aweiteka** [comment]
@mtrmac @runcom needs approval

**Comment by runcom** [comment]
lgtm

**Comment by mtrmac** [comment]
:+1:

**Pull request merged**